controllers/openstackclient_controller.go

```diff
@@ -603,7 +609,8 @@ func (r *OpenStackClientReconciler) podCreateOrUpdate(
 		return nil
 	})
 	if err != nil {
-		if common.IsForbiddenPodSpecChangeError(err) || k8s_errors.IsInvalid(err) {
+		var forbiddenPodSpecChangeErr *k8s_errors.StatusError
+		if errors.As(err, &forbiddenPodSpecChangeErr) || k8s_errors.IsInvalid(err) {
```

Comment:
will this be true for any status error, not just a Forbidden and Invalid?

Reply:
right its on any status error, will update it